base/doc/clsguide.tex

```diff
@@ -977,6 +977,30 @@ \subsection{Passing options around}
 handler, but this handler is not used for |landscape| because that option is
 explicitly declared.
 
+\subsection{Testing dates of other code}
+
+\begin{decl}
+  |\IfPackageAtLeast(TF)| \arg{package} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfClassAtLeast(TF)| \arg{class} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfFileAtLeast(TF)| \arg{file} \arg{date} \arg{true code} \arg{false code}
```

Comment:
Use the same-named placeholders to be consistent with docs of `\Provides...` and others
```suggestion
  |\IfPackageAtLeast(TF)| \arg{package-name} \arg{date} \arg{true code} \arg{false code}\\
  |\IfClassAtLeast(TF)| \arg{class-name} \arg{date} \arg{true code} \arg{false code}\\
  |\IfFileAtLeast(TF)| \arg{file-name} \arg{date} \arg{true code} \arg{false code}
```

Also, similar to the doc of `\ProvidesFile`, maybe repeat here that "the full filename, including the extension, must be given" for `<file-name>`.
https://github.com/latex3/latex2e/blob/1f18053be5081bbe8b4a45193d26e101eaeac4f6/base/doc/clsguide.tex#L718-L723